envs/example/ci-full-rhel/group_vars/all.yml

```diff
@@ -72,5 +75,12 @@ heat:
 ironic:
   enabled: False
 
+glance:
+  store_smart: false
+  store_file: true
+
+lvm:
+  enabled: true
+
 cinder:
-  enabled: False
```

Comment:
leave cinder false as lvm enablement was done in master only and we do not want to backport that.